dotCMS/src/integration-test/java/com/dotcms/vanityurl/business/VanityUrlAPITest.java

```diff
@@ -457,499 +401,249 @@ public void checkInvalidActionTest()
      */
     @Test
     @WrapInTransaction
-    public void removeVanityURLDefaultLanguaguefromCacheTest() throws DotDataException, DotSecurityException {
+    public void resolving_multilingual_vanities_with_default_language_unpublished() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)